src/interface/TransactionRequestsService-swagger.yaml

```diff
@@ -374,7 +374,8 @@ paths:
           The Accept header field indicates the version of the API the client
           would like the server to use.
         in: header
-        required: true
+        # TODO: fix mojaloop_simulator getSignedChallenge response first
```

Comment:
@sridharvoruganti there is a need to add `accept` header in the mocked response in mojaloop_simulator